Locales/Portuguese_br.json

```diff
@@ -0,0 +1,1059 @@
+	"   E assim essa aventura acaba. O maligno Sedgewick derrotado, a bela Geti morta nos seus pés. Pela primeira vez em memória, o reino reluz com a paz, mas sofre com a morte da sua amável princesa...",
+//Sandbox.as
+	"Welcome to King Arthur's Gold":"Bem vindo a King Arthur's Gold",
+	"This is a sandbox mode in which\n you are free to relax.\n\nPress ESC to see controls.\nHave Fun!":"Este é um modo para relaxar.\n\nPressione ESC para ver os controles.\nSe divirta!",
+//Sandbox gamemode.cfg
+	"Build whatever your heart desires. Respect other folk's creations!":"Construa o que quiser. Respeite a construção dos outros!",
+
+//// VOTES ////
+//VoteCore.as
+	"Reason: {REASON}"            :"Motivo: {REASON}",
+	"Cast by: {USER}"             :"Votado por: {USER}",
+	"[O] - Yes"                   :"[O] - Sim",
+	"[P] - No"                    :"[P] - Não",
+	"Ctrl+O Pass"                 :"Ctrl+O Branco",
+	"CTRL+P Cancel"               :"Ctrl+P Cancelar",
+	"Click to close ({TIMELEFT}s)":"Aperte para fechar({TIMELEFT}s)",
+	"Start a Vote":"Votação",
+	"Can't Start Vote":	"Não é possível votar",
+	"(Vote already in progress)":"(Votação em progresso)",
+	//vote kick menu
+	"Kick {USER}?":"Expulsar {USER}?",
+	"Kick":"Expulsar",
+	"Vote Kick":"Votar Expulsão",
+	"Griefer":"Vândalo",
+	"Hacker":"Hacker",
+	"Teamkiller":"Matador de equipe",
+	"Chat Spam":"Chat Spam",
+	"Non-Participation":"Não-Participação",
+	"Voting requires a {REQUIRED_MIN} min wait\nafter each started vote to\nprevent spamming/abuse.\n":"Voto prescisa de {REQUIRED_MIN} minutos de\nespera depois de outro voto\npara previnir abuso.\n",
+	"Vote to kick a player on your team\nout of the game.\n\n- use responsibly\n- report any abuse of this feature.\n\nTo Use:\n\n- select a reason from the\n     list (default is griefing).\n- select a name from the list.\n- everyone votes.\n":"Vote para expulsar um jogador do seu time\nda partida.\n\nSeja responsável.\nReporte qualquer abuso dessa função.\n\nPara usar:\n\nSelecione um motivo da\n     lista (padrão é vandalismo).\nSeleicone um nome da lista.\nTodo mundo vota.\n",
+	"Cannot kick {USER}":"Não é possível expulsar {USER}",
+	"You started a vote for\nthis person last time.\n\nSomeone else must start the vote.":"Você votou para\nexpulsar essa pessoa últiam vez.\n\nOutro deve começar o voto.",
+	"Kick {USER}":"Expulsar {USER}",
+	"Kicking {USER}":"Expulsando {USER}",
+	"Make sure you're voting to kick\nthe person you meant.\n":"Tenha certeza você está votando\na pessoa certa.",
+	"Yes, I'm sure":"Sim, tenho certeza.",
+	"(No-one available)":"(Não disponível)",
+	"Can't vote":"Não pode votar",
+	"You are now allowed to votekick\nplayers on this server\n":"Você não pode expulsar\njogadores desse servidor\n",
+	//vote map menu
+	"Next Map":"Próximo Mapa",
+	"Map Ruined":"Mapa Arruiando",
+	"Stalemate":"Empasse",
+	"Game Bugged":"Jogo bugado",
+	"Skip to next map?":"Pular para próximo map?",

```

Comment:
you missed one thing its ''Pular para o próximo mapa''

Reply:
E também tem o "mapa arruiando" que deveria ser "mapa arruinado"

Reply:
Vlw guys, arrumei <3
Se acharem mais coisa me avisem :3

Reply:
Also, já da pra usar a tradução no game se quiserem ir vendo e recomendando alguma mudança aushaush
É só jogar na pasta KAG/Base/Locale e colocar nas opções do jogo pra tradução Portuguese_br

Reply:
como que baixa a tradução memo kkk n manjo mt de github